### PR TITLE
Leader address is correctly set

### PR DESCRIPTION
**Pull request by paulh86**
- leader address is properly set when confirmation of leader is executed
- before the leader election a dummy entry is set as node address

### CODE REVIEW

File: src/main/java/com/espro/flink/consul/leader/ConsulLeaderLatch.java [hunk 1]
```diff
@@ -124,35 +120,41 @@ private void watch() {
 				LOG.error("Exception during leadership election", e);
 				// backoff
 				try {
-					Thread.sleep(waitTime * 1000);
+                    TimeUnit.SECONDS.sleep(waitTimeInSeconds);
 				} catch (InterruptedException ignored) {
-
+                    Thread.currentThread().interrupt();
```
Comment (kbialek) [inline src/main/java/com/espro/flink/consul/leader/ConsulLeaderLatch.java:125]: :+1:

**Review by kbialek — APPROVED** [review]
Nice contributions! Please resolve the merge conflict and merge it

**Comment by paulh86** [comment]
I am not able to merge this branch, because i have no write permissions to this repository.

**Comment by kbialek** [comment]
@paulh86 Invitation sent

**Pull request merged**